play-frontend-hmrc-play-30/src/test/scala/uk/gov/hmrc/govukfrontend/views/components/GovukBreadcrumbsSpec.scala

```diff
@@ -43,7 +45,9 @@ class GovukBreadcrumbsSpec extends TemplateUnitSpec[Breadcrumbs, GovukBreadcrumb
       )
       val output = component(params)
 
-      output.body should include("<div class=\"govuk-breadcrumbs\" role=\"navigation\" id=\"my-navigation\">")
+      output.body should include(
+        "<nav aria-label=\"Breadcrumb\" class=\"govuk-breadcrumbs\" role=\"navigation\" id=\"my-navigation\">"
+      )
```

Comment:
not sure why we have these tests really

Reply:
Yeah, I just updated the existing test here. I'll put a ticket up for tech debt to investigate whether we have redundant tests, as I think that could be a separate bit of work.

Reply:
Tech debt ticket raised to investigate all play-frontend-hmrc tests we think are redundant/unnecessary